src/modules/jackrack/CMakeLists.txt

```diff
@@ -1,8 +1,15 @@
-add_library(mltjack MODULE consumer_jack.c factory.c)
+add_library(mltjack MODULE factory.c)
 
 target_compile_options(mltjack PRIVATE ${MLT_COMPILE_OPTIONS})
 
-target_link_libraries(mltjack PRIVATE mlt Threads::Threads PkgConfig::jack)
+target_link_libraries(mltjack PRIVATE mlt Threads::Threads)
+
+if(TARGET PkgConfig::jack)
```

Comment:
How does one explicitly disable this dependency without removing the jack.pc file?

Reply:
I could add a option, but in what case is this useful?

Reply:
It is mainly useful for testing the build process in different configurations but also curious in case it is ever needed. Let's wait and see.